exoscale/resource_exoscale_compute.go

```diff
@@ -1101,3 +1134,23 @@ func prepareUserData(d *schema.ResourceData, meta interface{}, key string) (stri
 	}
 	return base64.StdEncoding.EncodeToString(byteUserData), false, nil
 }
+
+func reverseDNSApply(ctx context.Context, d *schema.ResourceData, client *egoscale.Client, id *egoscale.UUID) error {
```

Comment:
I'd be fine with this implementation, however if you want to improve it a bit more it looks like you don't really need to pass an `id` parameter to the `reverseDNSApply()` function since unless I'm mistaken you can infer it from the `d *schema.ResourceData` directly via `d.Id()` as done by the caller.

Reply:
fixed